src/useIsClient/useIsClient.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { renderHook } from '@testing-library/react'
+
+import useIsClient from './useIsClient'
+
+// TODO: currently don't know how to simulate hydration of hooks.
+describe('useIsClient()', () => {
+  it.skip('should be false when rendering on the server', async () => {
```

Comment:
Any help here would be appriciated

Reply:
For now, we haven't any clear direction about that

https://github.com/testing-library/react-testing-library/issues/1120